sat-plugin/src/main/java/org/openhab/tools/analysis/report/ReportMojo.java

```diff
@@ -1,37 +1,14 @@
 /**
```

Comment:
Was this header different by intention?

Reply:
Yes see https://github.com/openhab/static-code-analysis/blob/main/LICENSE

It's excluded from the check:

https://github.com/openhab/static-code-analysis/blob/7b5050dc554634e2cb0108aec041eec0c2eb80c6/tools/static-code-analysis/checkstyle/suppressions.xml#L8

Reply:
I think so, yes. It seems that [this code was copied](https://github.com/openhab/static-code-analysis/pull/427/files/8bd1abd56a7558b8d0c7122ad2dee8e650ec9158#diff-5895a1cf809a55b9aeb3485f43d31d3da4e121ad3c5001ebb0898611ca3bcdefR66-R71) and is licensed under BSD, so we should keep the copyright header in place.

Reply:
Alright. I reverted it in 76b0970 and excluded it from further formatting.